deployment/helm/node-feature-discovery/values.yaml

```
@@ -537,4 +537,5 @@ tls:
 
 prometheus:
   enable: false
+  interval: 10s
```

I'd suggest a small clarification on what interval are we configuring

```suggestion
  scrapeInterval: 10s
```

An alternative could be making deepening the hierarchy like `prometheus.podmonitor.interval` but I don't like that much, especially as we don't have other podmonitor settings